minecraft_servers/maybemc/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "server_name": "maybemc",
+  "nice_name": "MaybeMC",
+  "direct_ip": "maybemc.net",
+  "social": {
+    "web": "https://maybemc.net",
+    "web_shop": "https://shop.maybemc.net",
+    "web_support": "https://maybemc.net/forum",
+    "twitter": "maybemcnet",
+    "youtube": "https://www.youtube.com/@maybemcnetzwerk2759",
+    "instagram": "maybemcnet"
+  },
+  "discord": {
+    "server_id": 894672794774827061,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.